easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.0.0-foss-2016b-Python-3.5.2.eb

```diff
@@ -0,0 +1,67 @@
+# @author: 
+#      Marcel Schoengens (CSCS) 
+#      Guilherme Peretti-Pezzi (CSCS)
+#      Victor Holanda Rusu (CSCS)
+
+easyblock = "CmdCp"
+
+name = 'TensorFlow'
+version = '1.0.0'
+pyver = '3.5.2'
+versionsuffix = '-Python-%s' % pyver
+
+homepage = 'https://www.tensorflow.org/'
+description = """An open-source software library for Machine Intelligence."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+builddependencies = [
+    ('wheel', '0.30.0a0', '-Python-%(pyver)s'),
+    ('Bazel', '0.4.5'),
+]
+
+dependencies = [
+    ('Python', pyver),
+    ('SWIG', '3.0.10', '-Python-%(pyver)s'),
+]
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/tensorflow/tensorflow/archive/']
+
+
+# Patch to remove hardcoded paths for GCC and Binutils
+patches = [('tensorflow-v%(version)s-Python-%(pyver)s-foss.patch')]
+
+with_configure = False
```

Comment:
well, `CmdCp` derives from `MakeCp`, so it's taken into account, but the default value is already `False`:

```
$ eb -a -e CmdCp | grep with_configure
with_configure*         Run configure script before building [default: False]
```